Directory.Build.targets

```diff
@@ -6,13 +6,20 @@
                                 '$(IsTestUtilityProject)' != 'true'">true</IsSourceProject>
   </PropertyGroup>
 
+  <ItemDefinitionGroup>
+    <TargetPathWithTargetPlatformMoniker>
+      <IsPackable>$(IsPackable)</IsPackable>
+    </TargetPathWithTargetPlatformMoniker>
+  </ItemDefinitionGroup>
+
   <Import Project="$(RepositoryEngineeringDir)overrideispackable.targets" Condition="'$(OverrideIsPackableForNonSourceProjects)' == 'true' and '$(IsSourceProject)' != 'true'" />
   <Import Project="$(RepositoryEngineeringDir)resources.targets" />
   <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.Arcade.Sdk" />
 
   <Import Project="$(RepositoryEngineeringDir)testing\runsettings.targets" Condition="'$(IsTestProject)' == 'true'" />
   <Import Project="$(RepositoryEngineeringDir)placeholderpackaging.targets" Condition="'$(IsPlaceholderTargetFramework)' == 'true' and '$(IsCrossTargetingBuild)' != 'true'" />
   <Import Project="$(RepositoryEngineeringDir)packaging.targets" Condition="'$(IsPackable)' == 'true'" />
+  <Import Project="$(RepositoryEngineeringDir)servicingValidation.targets" Condition="'$(IsPackable)' != 'true' and '$(IsSourceProject)' == 'true'" />
```

Comment:
I would call this differently as this targets runs all the time, in main which technically is servicing all the time but we don't call it servicing. Some ideas: packageDependencyValidation.targets.